sdks/python/apache_beam/yaml/yaml_mapping.py

```diff
@@ -467,7 +467,9 @@ def expand(self, pcoll):
           break
       else:
         raise ValueError(
-            f"No field name matches one of {self._ERROR_FIELD_NAMES}")
+            'The input to this transform does not appear to be an error '
+            + "output.  Expected a schema'd input with a field named "
+            + ' or '.join(repr(fld) for fld in self._ERROR_FIELD_NAMES)
```

Comment:
```suggestion
            + ' or '.join(repr(fld) for fld in self._ERROR_FIELD_NAMES))
```

Probably this will still need formatting